src/browser/components/ExtraBar.jsx

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Row} from 'react-bootstrap';
+
+export default class ExtraBar extends React.Component {
+  handleBack = () => {
+    if (this.props.mattermostView) {
+      this.props.mattermostView.goBack();
+    }
+  }
+  render() {
+    let barClass = 'clear-mode';
+    if (!this.props.show) {
+      barClass = 'hidden';
+    } else if (this.props.darkMode) {
+      barClass = 'dark-mode';
+    }
+
+    return (
+      <Row
+        id={'extra-bar'}
+        className={barClass}
+      >
+        <div
+          className={'container-fluid'}
+          onClick={this.handleBack}
+        >
+          <span
+            id={'back_button_icon'}
+            className={'backIcon fa fa-1x fa-angle-left'}
+            title={'Back Icon'}
+          />
+          <span className={'backLabel'}>{'Back'}</span>
```

Comment:
We should use a more semantic `<button>` or tag for this and make sure keyboard navigation will work. See the `link` button example at the top of the React Bootstrap page: https://react-bootstrap.github.io/components/buttons/.